ui/design-system-core/src/components/ModerationAppErrorCard/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
```

Comment:
we should keep more generic components in the design system core, so either make it something like ErrorCard and make it more generic or implement it directly in the app